tests/foreman/ui/test_organization.py

```diff
@@ -380,42 +306,6 @@ def test_negative_update(self):
                         common_locators['name_haserror'])
                     self.assertIsNotNone(error)
 
-    @run_only_on('sat')
-    @skip_if_bug_open('bugzilla', 1321543)
-    @tier2
-    def test_positive_create_with_all_environment(self):
```

Comment:
removed as BZ is closed with WONTFIX resolution

Reply:
did you check functionality?

Reply:
Yes, it's not fixed (like in BZ description).

Reply:
I vote to not delete this test